spec/api/v4/users_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'swagger_helper'
+
+describe 'Users v4 API', swagger_doc: 'v4/swagger.json' do
+  path '/users/activate' do
+    post 'Authenticate user, request OTP, and get user information' do
+      tags 'User'
+      parameter name: :user, in: :body, schema: Api::V4::Schema.user_activate_request
+
+      before :each do
+        sms_notification_service = double(SmsNotificationService.new(nil, nil))
```

Comment:
Curious, what's the difference between this and `sms_notification_service = double`?

Reply:
This is a [verifying double](https://relishapp.com/rspec/rspec-mocks/docs/verifying-doubles)